.github/scripts/build-and-run-sanitizers.sh

```diff
@@ -4,35 +4,26 @@
 #
 # Copyright (C) 2019-2021  Kevin R. Croft <krcroft@gmail.com>
```

Comment:
> 50% change; suggest replacing w/ DOSBox Staging Team

Reply:
I mostly removed/renamed stuff and changed indentation - I think it's ok to keep it in :)

I intended 50% rule to work mostly for files that are currently covered by DOSBox Team copyright only (because it's weird when we give copyright to another team for code they don't want to merge).